src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -0,0 +1,119 @@
+import math
+import os
+from functools import partial
+from typing import Any, Mapping, Optional, Sequence, Tuple, Union
+
+import tiledb
+from tiledb.cloud.utils import logger
+
+
+def ingest(source: Union[Sequence[str], str],
```

Comment:
Can we support `source` being an s3 path being also a s3 path? For the user to give a path and say "ingest everything in it"?

Reply:
> Can we support source being an s3 path being also a s3 path

You mean an `s3` bucket?

Reply:
No I mean for the user to give a "folder", a "single file" or a "list of files". To support all 3 permutations.